src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int value = random.nextInt(100);
```

Comment:
use constant for 100
```suggestion
        int value = random.nextInt(100);
```